src/drivers/kmdf/queries/DeviceInitApi/test/fail/README.md

```diff
@@ -0,0 +1,20 @@
+---
+page_type: sample
+description: "Demonstrates how CodeQL can find errors in a KMDF driver."
+languages:
+- cpp
+products:
+- windows
+- windows-wdk
+---
+
+# SDV-FailDriver-KMDF
```

Comment:
Please make a todo to remove all references to SDV, including in file names and function names. They are throughout the code and unnecessarily complicate it, but I will not block on that.

Reply:
Thanks.  I will add a TODO, but FWIW my expectation is that once Abenezer merges his PR I'll create a new PR removing these full-size drivers and replacing them with code snippets that work with the testing script he and Jakob put together.

Reply:
I am very much looking forward to Nate leveraging the snippet unit test framework.  That will make unit test oh so much more compact and succinct.